assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -97,12 +97,11 @@ const EditorWizardModal = () => {
 		course: [ CourseDetailsStep, CourseUpgradeStep, CoursePatternsStep ],
 		lesson: [ LessonDetailsStep, LessonPatternsStep ],
 	};
-	const { postType } = useSelect( ( select ) => ( {
+	const { postType, senseiProExtension } = useSelect( ( select ) => ( {
 		postType: select( editorStore )?.getCurrentPostType(),
+		senseiProExtension: select( EXTENSIONS_STORE ).getSenseiProExtension(),
 	} ) );
```

Comment:
Sorry that I just noticed now, but one thing here is that we just need to get it one time, so we can use the [`deps`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data/#useselect) as `[]`.